### DISCUSSION TITLE
How to annotate expanded/extracted gene families?

### DISCUSSION THREAD

**Discussion by Ruiqi-CUB** [hahnlab/CAFE5#119] [Q&A] [answered]
Hi there! Thanks for the awesome tool!

I am using OrthoFinder output for CAFE and I would like to know the functions of the orthogroups (gene familiers) that are expanded or extracted.

I annotated each genomes used for OrthoFinder with eggnog mapper.

I am not sure how to annotate the gene family though: each orthogroup has serveral genes from different species, and their annotations are always not the same. How can I annotate each orthogroup? Should I get the function (for example, gene name) of each gene in that orthogroup, calculate the frequency, then pick the most frequent function?

Thanks
Ruiqi

**Comment by MegadogDede** [comment, marked as answer]
Combining with the result output by EggNog , the KEGG and GO Enrichment in "TBTools" software may be help.[https://geekdaxue.co/read/cjchen@hirv8i/pohup4](url)
